src/frontend/Errors.ml

```diff
@@ -1,34 +1,30 @@
 (** Setup of our compiler errors *)
 
 open Core_kernel
-open Middle
```

Comment:
What's bad about opening Middle? I don't see any ambiguity and editors are quick to resolve to the right place with goto-definition.

Reply:
Force of habit - I almost never open modules at the top level. I'll open and remove the qualifications